ui/src/app/base/components/SectionHeader/SectionHeader.tsx

```diff
@@ -69,8 +69,9 @@ const SectionHeader = <P,>({
             <h4
               className="section-header__title"
               data-test="section-header-title-spinner"
+              aria-label="loading"
             >
-              <Spinner text="Loading..." />
+              <Spinner text="Loading..." aria-hidden="true" />
```

Comment:
Are the changes to this file something that we should put into the `Spinner` component instead?

Reply:
Yes, we should spread `span` props on to the `Spinner` component because right now this `aria-hidden` prop isn't transferred to the rendered markup.

https://github.com/canonical-web-and-design/react-components/issues/651